src/main/java/com/microsoft/azure/functions/worker/Utilities.java

```diff
@@ -0,0 +1,10 @@
+package com.microsoft.azure.functions.worker;
+
+public class Utilities {
+    public static boolean isTrue(String value) {
+        if(value != null && value.toLowerCase().equals("true")) {
```

Comment:
Rename to `isLoadAppLibs` .  Make this method specific to checking if `FUNCTIONS_WORKER_JAVA_LOAD_APP_LIBS` is set to true or "1"

Reply:
I created as Utilities so we can use it anywhere in future. I believe much better to be more generic as the logic is not specific for FUNCTIONS_WORKER_JAVA_LOAD_APP_LIBS .

The file should name util my fault.

Reply:
having static methods that sepcific to checking environment is useful. Please see here for example pattern: https://github.com/Azure/azure-functions-host/blob/dev/src/WebJobs.Script/Environment/EnvironmentExtensions.cs also `isTrue` is very generic. I recommend making specific helpers.

Reply:
Good point I added a helper for that.